apps/google-analytics-4/frontend/src/components/config-screen/api-access/setup/SetupServiceAccountCard.tsx

```diff
@@ -155,7 +155,8 @@ export default function SetupServiceAccountCard(props: Props) {
           rel="noopener noreferrer">
           Google Cloud service account
         </TextLink>{' '}
-        for which you enable <i>read access</i> to your organization's Google Analytics data.
+        for which you enable "viewer" access to one of your organization's Google Analytics
+        properties.
       </Paragraph>
       <Paragraph marginBottom="none">
         After configuring the service account, you'll download a set of credentials that Contentful
```

Comment:
I would recommend changing this to:

```
With the service account configuration, you will create a new service account key file (JSON) that will enable you to access Google Analytics 4 data from within Contentful.
```

Reply:
might help connect the dots a bit more for folks in the GCP UI I think (specifically saying "service account key file" and "JSON" since they have to navigate through the UI making both of these distinguishing selections)

Reply:
Yes

Reply:
I think "you" is a bit weird here btw, but leaving it. The more pedantically correct wordings ("this app" or "users of this space environment") are unhelpfully more complicated.